bugsnag-plugin-android-exitinfo/src/test/java/com/bugsnag/android/TombstoneEnhancerTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.bugsnag.android
+
+import android.app.ApplicationExitInfo
+import com.bugsnag.android.internal.ImmutableConfig
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.mockito.junit.MockitoJUnitRunner
+import com.bugsnag.android.Thread as BugsnagThread
+
+@RunWith(MockitoJUnitRunner::class)
+internal class TombstoneEnhancerTest {
+
+    private val logger = mock(Logger::class.java)
+
+    private val tombstoneEnhancer = TombstoneEnhancer(logger)
+
+    private val file = this.javaClass.getResourceAsStream("/tombstone_01.pb")
```

Comment:
This should be moved into the test method, since an `InputStream` can only be read once.